dist/mods/ui/web/screens/voip/rtc.js

```diff
@@ -228,40 +234,50 @@ function stopSpeak(user, peer)
 		user: user,
 		volume: peer.speakingVolume,
 		isSpeaking: false
-	}
-	dew.notify("voip-speaking", speaker);
-	var index = $.inArray(user, speaking);
-	if(index != -1)
-		speaking.splice(index, 1);
-	
-	var thisUser = $("#" + user);
-	var index = thisUser.index();
-	if($("#speaking > p").length > 0)
-	{
-		$("#speaking > p").each(function(){
-			if($(this).index() < index)
-				return; //this user does not need to move
-			
-			y = $(this).height() * $(this).index() - $(this).height();
-			$(this).css({
-				'transform': 'translate(' + 0 + 'px, ' + y + 'px)',
+    }
+
+    dew.notify("voip-speaking", speaker);
+
+    dew.command('VoIP.SpeakingPlayerOnHUD').then(function (res) {
```

Comment:
In the event that someone changes the VoIP.SpeakingPlayerOnHUD variable from 0 to 1 while a player is speaking, the name of that player will not be removed from the cef overlay. 

The players currently speaking should be removed from the overlay when the variable is changed.